conda/recipes/conda_build_config.yaml

```diff
@@ -79,10 +79,7 @@ readline:
   - <8.2
 
 euphonic:
-  - 0.6.*
-
-importlib_resources:
-  - 5.12.0
+  - '>=1.2.1 <2.0'
```

Comment:
I think you need `,`
```bash
'>=1.2.1,<2.0'
```
https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html#package-match-specifications